plugins/zoom/icon_zoom/connection/connection.py

```diff
@@ -46,7 +47,20 @@ def connect(self, params):
     def test(self):
         try:
             # Return self to confirm API and key is working
+            self.zoom_api.authenticate()
             self.zoom_api.get_user("me")
             return {"success": True}
         except PluginException as e:
             raise ConnectionTestException(cause=e.cause, assistance=e.assistance, data=e)
+        except AuthenticationRetryLimitError:
+            raise ConnectionTestException(
+                cause="OAuth authentication retry limit was met.",
+                assistance="Ensure your OAuth connection credentials are valid. "
+                           "If running a large number of integrations with Zoom, consider "
+                           "increasing the OAuth authentication retry limit to accommodate.",
+            )
+        except AuthenticationError:
+            raise ConnectionTestException(
+                cause="The OAuth token credentials or JWT token provided in the connection configuration is invalid.",
+                assistance="Please verify the credentials are correct and try again."
```

Comment:
I see cause and assistance text hardcoded in many places. Can we create constants in util.py and use constants wherever applicable?